src/components/Navbar.jsx

```diff
@@ -1,17 +1,31 @@
 import React from 'react';
 import Modal from 'react-responsive-modal';
+import PropTypes from 'prop-types';
 import { Icon, Navbar as MaterialNavbar, NavItem } from 'react-materialize';
 import LoginContainer from '../containers/Auth/LoginContainer';
 import SignUpComponent from '../containers/Auth/SignUpContainer';
 import Dropdown from './Dropdown';
 import authUser from '../utils/authUser.util';
 
 class Navbar extends React.Component {
-  state = {
-    open: false,
-    openRegistration: false,
-    menuOpen: false,
-  };
+  constructor(props) {
```

Comment:
I thought we agreed on using ES6 syntax, and as such, we should not be using the constructor method

Reply:
@lukorito can explain this better, I will lias with him tomorrow